src/molecules/account-menu.module.css

```
@@ -0,0 +1,29 @@
+/* AccountMenu.module.css */
+.accountMenu {
+    background-color: var(--surface-container);
+    color: var(--on-surface);
+    box-shadow: 0 4px 8px var(--shadow);
```

```suggestion
    box-shadow: 2px 2px 8px color-mix(in srgb, currentcolor 25%, transparent);
```
このくらいの広がりと色味のほうが自然な影になりそうです.